galata/src/helpers/activity.ts

```diff
@@ -59,6 +59,29 @@ export class ActivityHelper {
     }
   }
 
+  /**
+   * Keyboard navigation to element using navigation key
+   *
+   * @param attributeValue value of selected widget attribute
+   * @param key navigation key to press
+   * @param attribute name of attribute selector
+   */
+
+  async keyToSidebar(
```

Comment:
Having a second look at this I could not understand what this helper does from reading the docstring alone; I needed to read the code to understand and have further request on it as this would become a part of public API and a headache to change in the future if needed:
- can you clarify the docstring that it will press the key until it reaches an element
- the name needs changing: the docstring mentions generic  "element" but function name includes "sidebar"; if it means to be generic, it should be named as such
- the key can be customized; but in fact is expected to be either a `Tab`, `ArrowDown`, `ArrowUp` or possibly a few others; maybe it would be worth replacing `string` with `'Tab' | 'ArrowDown' | 'ArrowUp'`?
- the `attribute` and `attributeValue` are too specific to make this helper useful in the general case; I would suggest to replace these two with a single `selector: string` argument and use `document.activeElement.matches(selector)` instead.
- there is an empty new line between doscting and function, it needs to be deleted

Reply:
Thanks for the suggestions, have updated